Scripts/RemediationScripts/Readme.md

```diff
@@ -1,4 +1,54 @@
-## Load remediation script to fix failed controls of Azure Tenant Security Solution - Step by Step
+# Bulk Remediation Script (BRS)
+
+## On this page:
+
+- [Overview](README.md#overview)
+- [BRS supported control list](README.md#brs_supported_control_list)
+- [How to use BRS](README.md#how_to_use_brs)
+
+-----------------
+
+## **Overview**
+Bulk remediation scripts can be used to remediate non-compliant resources/subscription for a control. These scripts are limited to the controls that have relatively lower risk to auto-remediate.
+
+## **BRS supported control list**
+|Control Id|Control Name|Link to BRS|Minimum permissions required to run the script|Supports managed identity based remediation**|Supports rollback?|
```

Comment:
The articulation has been very done (esp. the column headings). Could we **not** use a tabular view since it is difficult to read and scroll (left and right). A heading per control with sub-headings (and each section separated) should work. 

Heading should be "**List of controls with Bulk remediation script (BRS) support**" (instead of BRS supported control list)

Reply:
Done.